### ISSUE TITLE
[Feature Request] URL Filter (+ message about number of connections)

### ISSUE THREAD

**Issue by vassilyzaitsev** [jingyu9575/multithreaded-download-manager#193]
Man, I think if this download manager had a URL filter so we could set it up to automatically download from this or that website and the others to automatically be downloaded using the native it would be awesome.

Because quite often we face websites that allow and some that don't... and having this option would just save so many clicks and brain energy 😅😅

Something like: Always download using the browser downloader except in these cases: url list.
Or: Always use the extension except in these case: url list.

Also, I think the extension could check (if possible) the browser configs to see if the number of connections is correctly set up... I spend a lot of time not knowing I had to go to about:config and tweaking it up in order to download using more than 6 threads.

**Comment by fkohrt** [comment]
Yeah, it would be useful information that one has to adjust `network.http.max-persistent-connections-per-server` before utilizing more than six threads.

**Comment by revlisoft** [comment]
+1 for this.  I constantly have to disable the plugin for websites that don't support MT downloads like iCloud
